assets/app/views/settings.html

```diff
@@ -65,22 +65,53 @@
                 </div>
 
                 <div ng-repeat="quota in quotas" class="gutter-bottom">
-                  <div ng-if="quote.status.hard.cpu || quota.status.hard.memory">
-                    <h2>
-                      Quota <span ng-if="(quotas | hashSize) > 1">{{quota.metadata.name}}</span>
-                    </h2>
-                    <div ng-if="$first" class="help-block">{{quotaHelp}}</div>
+                  <h2>
+                    Quota <span ng-if="(quotas | hashSize) > 1">{{quota.metadata.name}}</span>
+                  </h2>
+                  <div ng-if="$first" class="help-block">{{quotaHelp}}</div>
+                  <dl ng-if="quota.spec.scopes.length" class="left">
+                    <dt>Scopes:</dt>
+                    <dd>
+                      <div ng-repeat="scope in quota.spec.scopes">
+                        {{scope | humanizeQuotaScope}}
+                        <span class="text-muted small">
+                          &mdash;
+                          <span ng-switch="scope" class="hide-ng-leave">
+                            <span ng-switch-when="Terminating">
+                              Matches all pods that have an active deadline.
+                            </span>
+                            <span ng-switch-when="NotTerminating">
+                              Matches all pods that do not have an active deadline.
+                            </span>
+                            <span ng-switch-when="BestEffort">
+                              Matches all pods that have best effort quality of service.
+                            </span>
+                            <span ng-switch-when="NotBestEffort">
+                              Matches all pods that do not have best effort quality of service.
+                            </span>
+                          </span>
+                        </span>
+                      </div>
+                    </dd>
+                  </dl>
+                  <div>
                     <div row mobile="column">
                       <div flex></div>
-                      <div column ng-if="quota.status.hard.cpu" class="center-block gutter-bottom">
```

Comment:
@derekwaynecarr can you confirm we don't need to handle `quota.status.hard.cpu` and `quota.status.hard.memory` values?